Wireframe/index.html

```diff
@@ -1,13 +1,112 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+
+<head>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0 shrink-to-fit=no" />
+  <link href="//fonts.googleapis.com/css?family=Roboto:400,500,300" rel="stylesheet" type="text/css" />
+  <link rel="shortcut icon" type="image/x-icon" href="img/github-logo-5F384D0265-seeklogo.com.png" />
+  <script src="https://kit.fontawesome.com/c0b874b18b.js" crossorigin="anonymous"></script>
+  <title>Git WireFrame</title>
+  <link rel="stylesheet" href="/Wireframe/css/style.css">
+</head>
+
+<body>
+  <header>
+    <h1>GIT Control Versions</h1>
+    <p>
+      Git is a free and open source distributed version control system designed to handle everything from small to very
+      large projects with speed and efficiency.
+    </p>
+  </header>
+  <main>
+    <section class="hero">
```

Comment:
I'm not sure section makes sense here. It's interesting -- I've seen loads and loads of PRs and everyone is grouping articles with section. Typically a section would go inside an article. https://html.spec.whatwg.org/#the-section-element

I can't see the semantic purpose of section here (or in any of the PRs) but I'm so curious as to how so many people have come to the same conclusion. I wonder if you know?